knative-operator/pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -51,6 +57,40 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	resources, err := kourier.Resources(ingressNamespace)
+	if err != nil {
+		return err
+	}
+	// Watch Kourier resources.
+	for _, u := range resources {
+		g := u.GroupVersionKind()
+		gvk := schema.GroupVersionKind{Group: g.Group, Kind: g.Kind, Version: g.Version}
+		u.SetGroupVersionKind(gvk)
```

Comment:
These three lines seem to do a redundant thing?

Reply:
Done.